recipes/simdutf/bld.bat

```diff
@@ -0,0 +1,4 @@
+mkdir build
+pushd build
+cmake %CMAKE_ARGS% -GNinja -DSIMDUTF_TOOLS=OFF -DBUILD_SHARED_LIBS=ON ..
```

Comment:
Please add a check to line 3 and 4 that the commands succeeded. CMD doesn't do this by default.